src/components/Characters/Fox/__tests__/FoxArtwork.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import FoxArtwork from '../FoxArtWork';
```

Comment:
Please separate external libs and internal imports with new line.

Reply:
Fixed in https://github.com/BuildIt-Poland/daily-routine/pull/35/commits/0de41235ec4270d7091be52f43d640119357dd80